quickstarts/microsoft.cdn/front-door-premium-storage-blobs-private-link/modules/storage.bicep

```diff
@@ -25,6 +25,9 @@ resource storageAccount 'Microsoft.Storage/storageAccounts@2022-05-01' = {
   }
   properties: {
     publicNetworkAccess: 'Disabled'
+    allowBlobPublicAccess: true
```

Comment:
why public?

Reply:
Front Door can't reach the blob container unless this is enabled. However, the network restriction ensures that it's not actually publicly accessible.

It's enabled like this by default - I just included it here to make it clear that this is explicitly intended.

Reply:
Ack - you may want to file an issue for templateAnalyzer, since they don't seem to factor in the network piece...

https://github.com/Azure/template-analyzer